packages/cozy-pouch-link/src/CozyPouchLink.js

```diff
@@ -97,13 +105,38 @@ class PouchLink extends CozyLink {
     this.client = client
   }
 
+  async migrateAdapter({ fromAdapter, toAdapter, url, plugins }) {
+    try {
+      for (const plugin of plugins) {
+        PouchDB.plugin(plugin)
+      }
+      const doctypes = getPersistedSyncedDoctypes()
+      for (const doctype of doctypes) {
+        const prefix = getPrefix(url)
+        const dbName = getDatabaseName(prefix, doctype)
+        const oldPouch = new PouchDB(dbName, {
+          adapter: fromAdapter,
+          location: 'default'
+        })
+        await migratePouch(oldPouch, toAdapter)
+        await oldPouch.close()
+        await oldPouch.destroy()
+      }
+      persistAdapterName('indexeddb')
+      // Reload page to benefit from new adapter
+      window.location.reload()
```

Comment:
Since this is the responsability of the app to do the display during the migration, it should also be the responsability of the app to know what to do after the migration?

Reply:
I think we mentioned together to do it through cozy-client, but I don't quite remember the reasoning. Anyway, it makes sense to me to do it on the app side.

Reply:
Fixed in 24cbbc9